DAE/backends/impala/parquet_io.py

```diff
@@ -218,17 +218,35 @@ def pedigree_parquet_schema():
         pa.field("status", pa.int8()),
         pa.field("role", pa.int32()),
         pa.field("sample_id", pa.string()),
+        pa.field("generated", pa.bool_()),
+        pa.field("layout", pa.string()),
     ]
 
     return pa.schema(fields)
 
 
+def add_missing_parquet_fields(pps, ped_df):
```

Comment:
probably this function should become 'private' and contain 'pedigree' in its name...